src/metpy/io/gempak.py

```diff
@@ -170,6 +168,10 @@ class DataSource(Enum):
 ])
 
 
+def convert_degC_to_K(val):
```

Comment:
This function needs to handle missing values (more often then not, -9999). This is why the tests failed as it just marched along with those values.